src/app/pages/Collectibles/CollectiblePage/components/Details.tsx

```diff
@@ -185,24 +194,26 @@ export const EvmDetails = memo<EvmDetailsProps>(({ network, accountPkh, assetSlu
         </div>
       </ChartListItem>
 
-      <ChartListItem title={<T id="tokenStandard" />}>
-        <p className={VALUE_CLASSNAME}>{displayStandard}</p>
-      </ChartListItem>
+      {displayStandard && (
+        <ChartListItem title={<T id="tokenStandard" />}>
+          <p className={VALUE_CLASSNAME}>{displayStandard}</p>
+        </ChartListItem>
+      )}
 
       <ChartListItem title={<T id="tokenId" />}>
```

Comment:
Please consider creating a component in this file for `ChartListItem` instances like this one